sdk/monitor/azure-monitor-ingestion/README.md

```diff
@@ -30,11 +64,12 @@ that you create in Log Analytics workspace. You can even extend the schema of bu
 ### Create the client
 
 An authenticated client is required to upload logs to Azure Monitor. The library includes both synchronous and asynchronous forms of the clients. To authenticate, 
-the following examples use `DefaultAzureCredentialBuilder` from the [com.azure:azure-identity](https://search.maven.org/artifact/com.azure/azure-identity) package.
+the following examples use `DefaultAzureCredentialBuilder` from the [com.azure:azure-identity](https://central.sonatype.com/artifact/com.azure/azure-identity/1.9.0-beta.2/versions) package.
```

Comment:
Is there a way we can avoid linking to a specific version of the package?

Reply:
Unfortunately, the new maven website doesn't allow us to point to a list of versions without using one of the version numbers in the link.

Reply:
Yeah, we are going to need tooling to keep these versions updated, but unfortunately Maven Central doesn't have any convenience URLs.